Makefile

```diff
@@ -39,7 +39,7 @@ style_tests:
 system_tests:
 	pipenv run behave system_tests/features # This will only run the system tests
 
-acceptance_tests: system_tests
+acceptance_tests:
 	pipenv run behave acceptance_tests/features # This will run the system and acceptance tests
```

Comment:
Need to update the comment and the readme if this is no longer going to run the system tests first